flytekit/annotated/base_task.py

```diff
@@ -387,7 +406,11 @@ def dispatch_execute(
                 py_type = self.get_type_for_output_var(k, v)
                 if isinstance(v, tuple):
                     raise AssertionError(f"Output({k}) in task{self.name} received a tuple {v}, instead of {py_type}")
-                literals[k] = TypeEngine.to_literal(exec_ctx, v, py_type, literal_type)
+                try:
+                    literals[k] = TypeEngine.to_literal(exec_ctx, v, py_type, literal_type)
+                except Exception as e:
+                    raise AssertionError(f"failed to convert return value for var {k}") from e
```

Comment:
should we attempt to print the value?

Reply:
i dont like printing the value, what if its a large string?